### ISSUE TITLE
ROC AUC metric

### ISSUE THREAD

**Issue by Neeraj-Natu** [keras-team/keras-contrib#286]
For now i need to write a callback function to use the ROC AUC metric from sklearn, can we add this a Keras  metric instead.
